eng/common/init-tools-native.ps1

```diff
@@ -44,6 +44,9 @@ Param (
   [string] $GlobalJsonFile
 )
 
+Set-StrictMode -Version 2.0
```

Comment:
My only concern with this (when it goes to Arcade) is that this would change current behavior, and although it's probably the right thing to do,  it wouldn't surprise me at all if someone already depends on the fact that failures don't cause build breaks.

Reply:
:\ yeah you're right this would be a breaking change. I'll revert this for now and file an issue.

Reply:
Filed https://github.com/dotnet/arcade/issues/3502 to deal with this.

Reply:
I misread and did not touch `install-tools.ps1`. Idk what's going on here but init-tools-native.ps1 [already has](https://github.com/dotnet/arcade/blob/master/eng/common/init-tools-native.ps1#L51-L52) these in it.